glue_lab/handlers.py

```diff
@@ -4,20 +4,27 @@
 from jupyter_server.utils import url_path_join
 import tornado
 
+from .glue_utils import get_advanced_links
+
 
 class RouteHandler(APIHandler):
     # The following decorator should be present on all verb methods (head, get, post,
     # patch, put, delete, options) to ensure only authorized user can request the
     # Jupyter server
     @tornado.web.authenticated
-    def get(self):
-        self.finish(json.dumps({"data": "This is /glue-lab/get_example endpoint!"}))
+    def get(self, path):
+        if path == "available-advanced-links":
+            self.finish(json.dumps({"data": get_advanced_links()}))
```

Comment:
I think we can cache the result of `get_advanced_links` instead of calling it on every request.